common.mk

```diff
@@ -20,7 +20,8 @@ HELP_COMPILATION_VARIABLES += \
 "   ENABLE_CUSTOM_FIRRTL_PASS = if set, enable custom firrtl passes (SFC lowers to LowFIRRTL & MFC converts to Verilog)" \
 "   ENABLE_YOSYS_FLOW         = if set, add compilation flags to enable the vlsi flow for yosys(tutorial flow)" \
 "   EXTRA_CHISEL_OPTIONS      = additional options to pass to the Chisel compiler" \
-"   EXTRA_FIRRTL_OPTIONS      = additional options to pass to the FIRRTL compiler"
+"   EXTRA_FIRRTL_OPTIONS      = additional options to pass to the FIRRTL compiler" \
+"   ASPECT_ARGS				  = additional aspect flows that will be run based on inputted pointers (such as UPF gen)"
```

Comment:
Isn't `ASPECTS` the main variable that needs to be set w/ a comma separated list? Also formatting of the spaces.